.zenodo.json

```diff
@@ -99,6 +113,11 @@
       "affiliation": "JetBrains",
       "type": "Other"
     },
+    {
+      "name": "Pantaleo, Felice",
+      "affiliation": "CERN",
+      "orcid": "0000-0003-3266-4357"
```

Comment:
The type others is missing and is required any name in contributors: `"type": "Other"`

Reply:
Done